lms/djangoapps/mobile_api/tests/test_milestones.py

```diff
@@ -85,28 +84,36 @@ def test_passed_entrance_exam(self):
 
     def _add_entrance_exam(self):
         """ Sets up entrance exam """
-        self.course.entrance_exam_enabled = True
-
-        self.entrance_exam = ItemFactory.create(  # pylint: disable=attribute-defined-outside-init
-            parent=self.course,
-            category="chapter",
-            display_name="Entrance Exam Chapter",
-            is_entrance_exam=True,
-            in_entrance_exam=True
-        )
-        self.problem_1 = ItemFactory.create(  # pylint: disable=attribute-defined-outside-init
-            parent=self.entrance_exam,
-            category='problem',
-            display_name="The Only Exam Problem",
-            graded=True,
-            in_entrance_exam=True
-        )
-
-        add_entrance_exam_milestone(self.course, self.entrance_exam)
-
-        self.course.entrance_exam_minimum_score_pct = 0.50
-        self.course.entrance_exam_id = unicode(self.entrance_exam.location)
-        modulestore().update_item(self.course, self.user.id)
+        with self.store.bulk_operations(self.course.id):
+            self.course.entrance_exam_enabled = True
+
+            self.entrance_exam = ItemFactory.create(  # pylint: disable=attribute-defined-outside-init
+                parent=self.course,
+                category="chapter",
+                display_name="Entrance Exam Chapter",
+                is_entrance_exam=True,
+                in_entrance_exam=True,
+            )
+            self.subsection_1 = ItemFactory.create(  # pylint: disable=attribute-defined-outside-init
```

Comment:
**Note to reviewers:** Needed to add a subsection in the test course hierarchy since grading of gated content is now based off of subsection grade changes.